docker.go

```diff
@@ -473,15 +477,16 @@ func newDockerImageSource(img, certPath string, tlsVerify bool) (*dockerImageSou
 		return nil, err
 	}
 	var tr *http.Transport
-	if certPath != "" {
```

Comment:
why did you refactor here? I don't like the  OR here - it was far more readable before this - I don't think this is fixing anything - if so could you please revert just this?